torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -205,6 +213,41 @@ void InplaceMKLDNNSubgraph(std::shared_ptr<Graph> graph) {
   }
 }
 
+Operation createUnaryOp(
+    std::function<void(at::Tensor output, at::Tensor input)> aten_op,
+    std::function<Tensor(Tensor)> fallback) {
+  return [aten_op, fallback](Stack* stack) {
+    auto a = pop(stack).toTensor();
+    if (a.numel() == 0) {
```

Comment:
if this is just to put numel() == 0 on the hot path instead of for correctness reasons is it worth it ?